providers/flagd/pom.xml

```diff
@@ -335,6 +335,38 @@
                                     </arguments>
                                 </configuration>
                             </execution>
+                            <execution>
+                                <id>copy-gherkin-flagd-rpc-caching.feature</id>
+                                <phase>validate</phase>
+                                <goals>
+                                    <goal>exec</goal>
+                                </goals>
+                                <configuration>
+                                    <!-- copy the feature spec we want to test into resources so them can be easily loaded -->
+                                    <!-- run: cp test-harness/features/flagd.feature src/test/resources/features/ -->
```

Comment:
Please update these comments so they are accurate, or just remove them.

Reply:
do we need those comments? should we myabe remove them?

Reply:
That's OK with me.